### PR TITLE
US02-02 Bloc de text page home

### PR DESCRIPTION
**Pull request by NEL45**

### CODE REVIEW

File: public/assets/css/welcome.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    color: #fff;
+    width: 100%;
+    height: 100vh;
+    background-color: rgba(0, 0, 0, 0.5);
+}
+.principal{
+    --letter-spacing: 5px;
+    font-family: var(--title-font);
+    --light: #FFFFFF;
+    left: 50%;
```
Comment (bouboumael) [inline public/assets/css/welcome.css:24]: many magic number and attention you have 2 pr in one.

File: src/View/User/Includes/_welcome_background_slideshow.html.twig [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{# User/Includes/_welcome_background_slideshow.html.twig #}
+
+<div class="slider">
+        <div class="load">
+        </div>
+        <div class="content">
+            <div class="principal">
+            <h1>lorem ipsum</h1>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing.</p>
+                {% include 'User/Includes/_welcome_card.html.twig' %}
```
Comment (bouboumael) [inline src/View/User/Includes/_welcome_background_slideshow.html.twig:10]: Is it really necessary to include another twig? the text must change?

File: src/View/User/Includes/_welcome_card.html.twig [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{# User/Includes/_welcome_card.html.twig #}
+
+<div class="card" style="width: 18rem;">
```
Comment (bouboumael) [inline src/View/User/Includes/_welcome_card.html.twig:3]: With the top comment

**Review by bouboumael — CHANGES_REQUESTED** [review]

File: public/assets/css/welcome.css (hunk 1 above)
Comment (NEL45) [reply public/assets/css/welcome.css:24]: comment applied

File: src/View/User/Includes/_welcome_background_slideshow.html.twig (hunk 2 above)
Comment (NEL45) [reply src/View/User/Includes/_welcome_background_slideshow.html.twig:10]: we created 2 distinct elements at the start in our user stories

File: src/View/User/Includes/_welcome_card.html.twig (hunk 3 above)
Comment (NEL45) [reply src/View/User/Includes/_welcome_card.html.twig:3]: we created 2 distinct elements at the start in our user stories

**Pull request closed without merging**